linter_exclusions.yml

```diff
@@ -73,6 +73,9 @@ aks create:
     workspace_resource_id:
       rule_exclusions:
       - option_length_too_long
+    enable_sgx_quote_helper:
```

Comment:
@MyronFanQiu , could you please help review this inter rule part ?

Reply:
@bowang-666 Is it possible to add extra short option with this one together? It's not good to just ignore this linter rule

Reply:
@MyronFanQiu This option will be mostly internal used. I don't think it's worth giving a short option since usage will be pretty low. Also it's hard to find one, only closest and not used I can find is "-q" but that confuses with --quit.

Reply:
@bowang-666 well. Please add exclusions in your extension instead of the global exclusion. https://github.com/Azure/azure-cli-extensions/blob/master/src/spring-cloud/linter_exclusions.yml

Reply:
Done